src/parsers/pm_io/data.jl

```diff
@@ -61,9 +61,10 @@ function calc_theta_delta_bounds(data::Dict{String,Any})
     end
 
     if haskey(data, "conductors")
-        amin = MultiConductorVector(angle_min)
-        amax = MultiConductorVector(angle_max)
-        return amin, amax
+        @error("Multiconductor Not Supported in PowerSystems")
```

Comment:
This makes the function return nothing.  The calling code will probably fail with a strange error message.  Do you want to use error() instead?